tests/pipelines/test_sklearn_classifier.py

```diff
@@ -0,0 +1,124 @@
+# flake8: noqa
+from functools import partial
+import os
+
+from pytest import mark
+import torch
+from torch.optim import Adam
+from torch.utils.data import DataLoader
+
+from catalyst import data, dl
+from catalyst.contrib import datasets, models, nn
+from catalyst.data.transforms import Compose, Normalize, ToTensor
+from catalyst.settings import SETTINGS
+
+if SETTINGS.ml_required:
+    from sklearn.ensemble import RandomForestClassifier
+    from sklearn.linear_model import LogisticRegression
+    from sklearn.neighbors import KNeighborsClassifier
+    from sklearn.svm import SVC
+
+TRAIN_EPOCH = 30
+LR = 0.01
+RANDOM_STATE = 42
+
+
+def train_experiment(device, engine=None):
+    from catalyst import utils
+
+    utils.set_global_seed(RANDOM_STATE)
+    # 1. train, valid and test loaders
+    transforms = Compose([ToTensor(), Normalize((0.1307,), (0.3081,))])
+
+    train_dataset = datasets.MnistMLDataset(
+        split=11, root=os.getcwd(), download=True, transform=transforms
+    )
+    sampler = data.BalanceBatchSampler(labels=train_dataset.get_labels(), p=5, k=60)
+    train_loader = DataLoader(
+        dataset=train_dataset, sampler=sampler, batch_size=sampler.batch_size
+    )
+
+    valid_dataset = datasets.MNIST(
+        root=os.getcwd(), transform=transforms, train=True, download=True
+    )
+    valid_loader = DataLoader(dataset=valid_dataset, batch_size=512)
+
+    test_dataset = datasets.MNIST(
+        root=os.getcwd(), transform=transforms, train=False, download=True
+    )
+    test_loader = DataLoader(dataset=test_dataset, batch_size=512)
```

Comment:
```suggestion
    train_dataset = datasets.MNIST(
        root=os.getcwd(), transform=transforms, train=True, download=True
    )
    train_loader = DataLoader(
        dataset=train_dataset, batch_size=128
    )

    valid_dataset = datasets.MNIST(
        root=os.getcwd(), transform=transforms, train=False, download=True
    )
    valid_loader = DataLoader(dataset=valid_dataset, batch_size=128)
```
let's make it simple?

Reply:
+ `test_loader = valid_loader`